internal/handlers/handler_test.go

```diff
@@ -80,6 +80,90 @@ func handle(ctx context.Context, t *testing.T, handler *Handler, req *types.Docu
 	return actual
 }
 
+func TestFindArray(t *testing.T) {
+	t.Parallel()
+	ctx, handler, pool := setup(t, nil)
+	schema := testutil.Schema(ctx, t, pool)
```

Comment:
This function creates a test-specific PostgreSQL schema / MongoDB database and drops it if the test passes. There is no need to do the manual cleanup with `cleanArrayTestData`.

Reply:
Got it, `cleanArrayTestData` is removed, thank you.